ynr/apps/popolo/api/next/filters.py

```diff
@@ -0,0 +1,42 @@
+from django_filters import filterset, filters
+
+from popolo.models import Organization, Post, Membership
+
+
+class OrganizationFilterMixin(filterset.FilterSet):
+    class Meta:
+        model = Organization
+        fields = ["last_updated"]
+
+    last_updated = filters.DateTimeFilter(
+        field_name="modified",
+        lookup_expr="gt",
+        label="Last updated",
+        help_text="An ISO datetime",
+    )
+
+
+class PostFilterMixin(filterset.FilterSet):
```

Comment:
Both these can be removed as neither the `Post` or `Membership` model have a viewset/api endpoint, so there is no way for us to filter them - they are returned as part of a parent API response instead (e.g. People lists their Memberships but we dont filter directly on the membership). Sorry I know this change came from me originally as I incorrectly said we should add filtering everywhere 🤦

Reply:
No problem, I'll remove it.